BUILD.bazel

```diff
@@ -26,6 +26,7 @@ filegroup(
     "protobufjs",
     "protractor",
     "reflect-metadata",
+    "rxjs",
```

Comment:
rxjs is actually not _supposed_ to be here because it has its own bazel rules. Can you share the error you were getting?

Reply:
I don't have the exact error, but it was something along the lines of "Can't find module `rxjs/index`".

Reply:
Chatted w/ Alex, this seems like an issue w/ the ngsummary files being generated, we're figuring out the right way for core to fix this